nextmv/nextmv/cloud/application.py

```diff
@@ -551,6 +556,30 @@ def delete_secrets_collection(self, secrets_collection_id: str) -> None:
             endpoint=f"{self.endpoint}/secrets/{secrets_collection_id}",
         )
 
+    def delete_ensemble_definition(self, ensemble_definition_id: str) -> None:
```

Comment:
Nit: we try to organize the methods in this class alphabetically, so this one should be above `delete_scenario_test`.